lib/logitech_receiver/base.py

```diff
@@ -189,6 +197,15 @@ def read(handle, timeout=DEFAULT_TIMEOUT):
 		return reply[1:]
 
 
+# sanity checks on  message and report incorrect sizes when debugging
+def debug_message(data) :
+	assert isinstance(data, bytes), (repr(data), type(data))
+	report_id = ord(data[:1])
+	assert ((report_id & 0xF0 == 0) or
+		(report_id in report_length and len(data) == report_length[report_id])), \
```

Comment:
Simplify: `report_lengths.get(report_id) == len(data)`
None won't match either. And please do rename the variable :)

Reply:
Ok, report_lengths it is.

Reply:
Done